test/e2e/component-properties-table.spec.js

```diff
@@ -0,0 +1,24 @@
+import { test, expect } from '@playwright/test';
+
+import { readdirSync } from 'fs';
+
+const componentFilenames = readdirSync('./docs/components')
+	.filter(i => i.endsWith('svelte.html'))
+	.filter(
+		i =>
+			![
+				'AreaStacked.svelte.html',
+				'BarStacked.svelte.html',
+				'ColumnStacked.svelte.html',
+				'GroupLabels.html.svelte.html'
+			].includes(i)
```

Comment:
My only thought is I wonder if we can auto-detect this but this isn't necessary to solve now.

Reply:
That's a good idea to future-proof that. I'll take a look. Should be possible somehow to skip the test if there is no match for `#params-table`.